src/resources/Abathur.ts

```diff
@@ -62,6 +62,10 @@ export const baseStockAmounts: { [key: string]: number } = {
 	[RESOURCE_HYDROGEN] : 5000
 };
 
+export function stockAmount(resource: ResourceConstant): number {
```

Comment:
I don't like exporting free-floating functions from module files other than random stuff like in utils.ts. Would prefer this to be a static method on abathur